apiserver/facades/client/modelconfig/modelconfig_test.go

```diff
@@ -85,9 +86,10 @@ func (s *modelconfigSuite) getAPI(c *gc.C) (*modelconfig.ModelConfigAPI, *gomock
 	).AnyTimes()
 
 	modelID := modeltesting.GenModelUUID(c)
+	s.controllerTag = names.NewControllerTag(coretesting.ControllerTag.Id())
```

Comment:
coretesting ControellerTag is a tag already
`s.controllerTag = coretesting.ControllerTag`

If we are using the tag from core testing, there's no need to assign it to a struct attribute
The better option is to keep the struct attribute and use `uuid.MustNewUUID().String()` to give it a value